snippets/olm-psa-opm-catalog-source-restricted-known-issue.adoc

```diff
@@ -0,0 +1,25 @@
+// Text snippet included in the following assemblies:
+//
+// * 
+//
+// Text snippet included in the following modules:
+//
+// * olm-catalogsource.adoc
+// * olm-creating-catalog-from-index
+
+:_content-type: SNIPPET
+
+There is a known issue in the `opm` CLI tool that prevents SQLite-based catalog source pods from reaching a ready state under restricted pod security enforcement. (link:https://issues.redhat.com/browse/OCPBUGS-122[*OCPBUGS-122*])
+
+Pod security admission was introduced in Kubernetes 1.24 to ensure pod security standards. In {product-title} {product-version}, namespaces do not have restricted pod security enforcement by default and the default security mode is set to `legacy`.
+
+It is planned that all namespaces will default to restricted pod security standard enforcement in a future release of {product-title}. It is recommended that you take action now to ensure your catalog sources run under restricted pod security enforcement. If you do not take action to run your catalog sources under restricted enforcement, your catalog sources might not run.
```

Comment:
`your catalog sources might not run.` -> you mean might not run in the future release, not 4.12, right?

Reply:
correct